### PR TITLE
docs: second lol ppp

### LINKED ISSUE #18: Create second PPP
https://makers.bolt.fun/story/ppps-week-2--219

### PR DESCRIPTION
**Pull request by holzeis**
resolves #18

### CODE REVIEW

**Review by Restioson — APPROVED** [review]
thanks!

File: ppp/iteration_2_24102022.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+### Plans 📆
+
+- Reiterate on the wireframes and polish them. (Hoping for feedback from the community 🤞)
```
Comment (klochowicz) [inline ppp/iteration_2_24102022.md:3]: do you mean 'create full-fledged mock-ups' by "polishing them"?

**Review by klochowicz — APPROVED** [review]
thanks for writing this up.

File: ppp/iteration_2_24102022.md (hunk 1 above)
Comment (luckysori) [inline ppp/iteration_2_24102022.md:3]: 🔧 I associate "reiterating" with saying something multiple times, although it's not wrong according to dictionaries. 

```suggestion
- Iterate on the wireframes and polish them. (Hoping for feedback from the community 🤞)
```

File: ppp/iteration_2_24102022.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- Investigating ldk integration to the flutter mobile app. A little bit prototyping. 🚀
```
Comment (luckysori) [inline ppp/iteration_2_24102022.md:4]: 🔧 Just some tweaks

```suggestion
- Investigate LDK integration with the Flutter mobile app. A little bit of prototyping 🚀
```

File: ppp/iteration_2_24102022.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- Write up a post about adding dlc to lightning channels. 💪
```
Comment (luckysori) [inline ppp/iteration_2_24102022.md:5]: 🔧 

```suggestion
- Write a post about adding DLCs to Lightning channels 💪
```

File: ppp/iteration_2_24102022.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+### Progress ✅
```
Comment (luckysori) [inline ppp/iteration_2_24102022.md:7]: ❓ Can we use hyperlinks rather than explicit URLs?

File: ppp/iteration_2_24102022.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- Decided on a mobile technology stack using flutter and rust. https://github.com/itchysats/10101/discussions/5
+- Created initial mobile wallet project using flutter, rust and flutter_rust_bridge. https://github.com/itchysats/10101/issues/9
+- Created wireframes of the user journey for opening and closing a non-perpetual CFD position. (TODO: add link to post here!)
+- Wrote bolt.fun post about the first draft of the mobile user journey opening and closing non-perpetual CFD positions.
+- Decided on a design approach for integrating dlc and lightning channels. https://github.com/itchysats/10101/issues/8
```
Comment (luckysori) [inline ppp/iteration_2_24102022.md:13]: 🔧 

```suggestion
- Decided on a design approach for integrating DLCs into Lightning channels. https://github.com/itchysats/10101/issues/8
```

**Review by luckysori — APPROVED** [review]
Thanks for writing this! Just some minor edits.

File: ppp/iteration_2_24102022.md (hunk 1 above)
Comment (holzeis) [reply ppp/iteration_2_24102022.md:3]: do we need to be specific? 🙈 I meant getting it to a point where we feel comfortable implementing it.

File: ppp/iteration_2_24102022.md (hunk 4 above)
Comment (holzeis) [reply ppp/iteration_2_24102022.md:7]: I don't think so, but I can't test unfortunately as I can't find the write story button on bolt.fun anymore 🙈

**Pull request merged**